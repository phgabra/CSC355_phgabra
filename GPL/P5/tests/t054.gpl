// use the floor() function from the math library
int a = floor(1.2);
int b = floor(-1.2);
int c = floor(-2);
int d = floor(-2.5);
int e = floor(-1.9999);
int f = floor(-2.0001);
