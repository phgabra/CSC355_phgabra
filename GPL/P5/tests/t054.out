gpl.cpp::main()
  input file(tests/t054.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
int a = 1
int b = -2
int c = -2
int d = -3
int e = -2
int f = -3
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
